[WF] add GovCloud-specific caution for Connections third-party data (#34196)
DataDog/documentation: content/en/actions/connections/_index.md

disable_toc: false
---

{{< site-region region="us,us3,us5,eu,ap1,ap2" >}}

> <div class="alert alert-caution">Actions, Workflows, and Apps can use Connections that may send Customer Data to third-party services.</div>

Reviewer: ```suggestion
<div class="alert alert-caution">Actions, Workflows, and Apps might use Connections that send customer data to third-party services.</div>
```

{{< /site-region >}}

{{< site-region region="gov" >}}

> <div class="alert alert-caution">Actions, Workflows, and Apps can use Connections that may send Customer Data to third-party services outside the Datadog for Government region. See the <a href="https://trust.datadoghq.com">Trust Center</a> for instructions to obtain the Customer responsibility matrix and additional information about the Datadog for Government region.</div>

Reviewer: ```suggestion
<div class="alert alert-caution">Actions, Workflows, and Apps might use Connections that send customer data to third-party services outside the Datadog for Government region. See <a href="https://trust.datadoghq.com">Trust Center</a> for instructions to obtain the customer responsibility matrix and additional information about the Datadog for Government region.</div>
```
